src/migrate/Migrator.js

```diff
@@ -97,7 +93,7 @@ export default class Migrator {
         .tap(validateMigrationList)
         .then((val) => this._getLastBatch(val))
         .then((migrations) => {
-          return this._runBatch(map(migrations, 'name'), 'down');
+          return this._runBatch(migrations, 'down');
```

Comment:
Was _runBatch somehow changed to allow to take migration list as an input instead of migration names?

Reply:
_runBatch per se no, it merely passes migrations param to getNewMigrations method which does expect the migration list.

Reply:
This change was needed because batch execution needs to be aware of directories as well.